src/client/websocket/packets/handlers/GuildMembersChunk.js

```diff
@@ -10,10 +9,9 @@ class GuildMembersChunkHandler extends AbstractHandler {
     const guild = client.guilds.get(data.guild_id);
     if (!guild) return;
 
-    const members = data.members.map(member => guild._addMember(member, false));
+    const members = new Collection(data.members.map(member => [member.user.id, guild._addMember(member, false)]));
 
-    guild._checkChunks();
-    client.emit(Constants.Events.GUILD_MEMBERS_CHUNK, members);
+    client.emit(Constants.Events.GUILD_MEMBERS_CHUNK, members, guild);
```

Comment:
The guild parameter was intentionally removed in v10, as it's easily available on any of the members.